networkx/algorithms/dag.py

```diff
@@ -628,9 +629,9 @@ def is_aperiodic(G):
 
     Notes
     -----
-    This uses the method outlined in [1]_, which runs in $O(m)$ time
-    given $m$ edges in `G`. Note that a graph is not aperiodic if it is
-    acyclic as every integer trivial divides length 0 cycles.
+    This uses the method outlined in [1]_, which runs in $O(n + m)$ time
```

Comment:
Are you basing this complexity description on the paper, or on your own examination, or on another source?

Reply:
The paper claims that the complexity is "O(n+m) = O(m)". The reason why it omits n is that it assumes the graph to be strongly connected.